include/revng/PipelineDescriptor/Analysis.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+/* TUPLE-TREE-YAML
+name: Analysis
+doc: Data structure representing an analysis
+type: struct
+fields:
+  - name: Name
+    doc: Analysis' name
+    type: string
+  - name: Arguments
+    doc: Analysis' arguments
```

Comment:
1. "Analysis' argument names"?
2. Don't arguments have a type too?

Reply:
1. The vector contains `AnalysisArgument`, which contains the name and type
2. See (1)

Reply:
https://github.com/revng/revng/pull/318#discussion_r1302856782